values.yaml

```diff
@@ -91,6 +91,7 @@ secrets:
 #  region: us-east-1
 #  regionEndpoint: s3.us-east-1.amazonaws.com
 #  bucket: my-bucket
+#  rootdirectory: my-docker
```

Comment:
It would be helpful for the example to illustrate this value as a path, to help the reader understand where slashes should be.

Which of the following is correct if I want object `foo` to become `object/prefix/foo`?

 • `object/prefix`
 • `/object/prefix`
 • `object/prefix/`
 • `/object/prefix/`

`my-docker` is not descriptive enough to convey confidence that it is conveying a path. Including a slash in the middle helps, and at the beginning and/or end as necessary. Having the example value itself convey through the words it uses that it is, unambiguously, an object prefix, further empowers the reader to identify the correct form for their use case.

Reply:
Hello @canterberry, thank you very much for quick reply. 
I have tested with all the cases you present, and the propagation works in all the cases you present. I agree with update of example so I have set to ```/object/prefix``` because it seems to me, it may be more descriptive.